package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@maplibre/maplibre-gl-geocoder",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
I think you need to run `npm install` to update the version in `package-lock.json` when you bump the version in `package.json`.